assets/js/modules/analytics/components/module/ModuleOverviewWidget/SiteStats.js

```diff
@@ -65,7 +65,7 @@ export default function SiteStats( { selectedStat, report } ) {
 			<Row>
 				<Cell size={ 12 }>
 					<GoogleChart
-						chartType="line"
+						chartType="LineChart"
```

Comment:
Seems like we need to add more props to this chart because its loading state doesn't look right:
![Screenshot from 2021-04-23 17-35-11](https://user-images.githubusercontent.com/873049/115887414-947a7f80-a45a-11eb-85aa-4ebbff16a273.png)

Plus, it looks like this chart stopped showing previous data:
![Screenshot from 2021-04-23 17-35-22](https://user-images.githubusercontent.com/873049/115887905-0d79d700-a45b-11eb-8c27-f5a851bf61b1.png)